source/release-notes.txt

```diff
@@ -42,7 +42,7 @@ Bug Fixes:
 
 - BSON Transpilers: Account for bson Decimal128 validation changes.
 
-- Make SSH tunnel use Socks5.
+- Make SSH tunnel use Socks5. You can connect to replica sets using Socks5.
```

Comment:
I'm not sure that users will be using Socks5. Instead, I think this is something that Compass enables under the hood that enables users to connect to replica sets using SSH. Therefore, we might want to change this sentence to something like:

"You can now connect to replica sets (and sharded clusters? not sure if this is true. Need confirmation from tech) using an SSH tunnel."

Reply:
Done and confirmed shared clusters with tech.